src/components/vm/vmReplaceSpiceDialog.jsx

```diff
@@ -32,15 +35,24 @@ import { domainReplaceSpice } from '../../libvirtApi/domain.js';
 
 const _ = cockpit.gettext;
 
-export const ReplaceSpiceDialog = ({ vmName, vmId, connectionName, vmRunning }) => {
+export const ReplaceSpiceDialog = ({ vm, vms }) => {
+    const spiceVMs = vms?.filter(vm => vm.inactiveXML?.hasSpice);
+    // sort spicVMs by name, put the current VM first
+    spiceVMs?.sort((a, b) => (a.name === vm.name) ? -1 : (b.name === vm.name) ? 1 : a.name.localeCompare(b.name));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1416-20240203-084715-a2627c26-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)